test/promote_literals_test.cpp

```diff
@@ -26,6 +26,7 @@
 #include <migraphx/program.hpp>
 #include <migraphx/dead_code_elimination.hpp>
 #include <migraphx/eliminate_common_subexpression.hpp>
+#include <migraphx/instruction.hpp>
```

Comment:
Is this being used ?

Reply:
Yes, compiler on Windows complains on incomplete type `instruction`.